go/libkb/interfaces.go

```diff
@@ -620,6 +621,9 @@ type Stellar interface {
 	CreateWalletSoft(context.Context)
 	Upkeep(context.Context) error
 	OnLogout()
+
+	SetServerDefinitions(context.Context, stellar1.StellarServerDefinitions) error
+	GetServerDefinitions(context.Context) (stellar1.StellarServerDefinitions, error)
```

Comment:
SetServerDefinitions shouldn't need to be exported. If there was a checked type cast to `*Stellar` in `RefreshServerConfig` that would be fine.

Reply:
makes sense, thanks!